polysemy-RandomFu/polysemy-RandomFu.cabal

```diff
@@ -70,4 +70,5 @@ test-suite polysemy-RandomFu-test
     , random-fu >=0.2.5.0 && <0.3.0.0
     , random-source >=0.3 && <0.4.0.0
     , text >=1.1.0.0 && <1.3.0.0
+    , vector
```

Comment:
Yeah, put bounds here as well.  And an upper bound too.  I've been doing that since it's good form though I think I will regret the maintenance headache it will bring...